website/docs/guides/release-notes.html.markdown

```diff
@@ -12,11 +12,40 @@ Track all Aviatrix Terraform provider's releases. New resources, features, and b
 
 ---
 
-``Last updated: R2.14.1 (UserConnect-5.4.1232)``
+``Last updated: R2.15 (UserConnect-6.0)``
 
 
 ---
 
+## 2.15.0
+### Notes:
+- Supported Controller version: **UserConnect-6.0**
+- Supported Terraform version: **v0.12.x**
+
+### Features:
+1. New data sources:
+  - **data_source_aviatrix_firewall**
+  - **data_source_vpc_tracker**
+2. Implemented support for the option to manage attachment on either **aviatrix_vpn_profile** or **aviatrix_vpn_user** using ``manage_user_attachment`` (and ``profiles`` for the user)
+3. Implemented support for ``action`` under domain_names filters for **aviatrix_fqdn**
+4. Implemented support for adding VPN users under GeoVPN workflow
+5. Implemented support for specifying ``ha_peering_subnet`` for GCP **aviatrix_gateway**
+6. Implemented support for specifying ``ha_subnet`` for GCP **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**
+7. Implemented support for ``enable_ikev2`` for **aviatrix_site2cloud**
+
+### Enhancements:
+1. Updated **aviatrix_site2cloud**'s``tunnel_type`` to support "policy" and "route"-based options in Controller 6.0
+2. Added ``route_tables`` and ``route_tables_filter`` in **data_source_vpc**
```

Comment:
```
aviatrix_vpc
```
?

Reply:
Good point. Didnt originally catch that in Tim's changelog

Reply:
Same.